.travis.yml

```diff
@@ -1,3 +1,16 @@
+addons:
+  artifacts:
+    debug: true
+    s3_region: "us-west-1"
+    target_paths:
+      - /${TRAVIS_REPO_SLUG}/
+
+env:
+  global:
+  - secure: EUSicS1mBOImgoYHBZJ2ZE0xwNMAkGRYUDetvxcKk0EQrOii1Z/EuYC1IN7mwiI6Ybnm3b2xq51bq/Ro3iQNi+iylA0r20mEjRsSqb8cqy8hXRnl4f7ZI3qUorUvEmsJueve3U2MLu5eYJdSNPbqgdavZDZZxiZIdbHAfA80KMp1RmgeqxkCY9wTfrYEqCtTGEJMJtCHXoDJ0cdYEVDnHQYZrF9SPkkMqmwgfFvHBGcyTPMsRZ9JpggxJltXJFTlHBNn/wLfGPiMsJlNx0oS7SmoZ08W72h2QKBZX5bVZxoNmMeQCZ0FlSHeq8yTGJ5xM8ig6YRzkuK5JiHkTaY9rVQMoU2ozMll9Bb5/A07XDXoutH1/QC1jvvT9eMBhon3uZ7e3qyUiRgCpFWrOW0UXKQqF2hhGWT965mBB5pxWOgGZjxtVIuKpFwMbsPIss44OuzlaFis5YRsKyceBAM+g8gkwxB8r8AT0A6g4jY4fmvrUr0qyaldlcr7LafGvAa73uJHoTUruR2AZQlJmLpO9p5xbuwId8L0alApqoaaR3vPx6LPtl/hTBr+KScs/I8PjdiEaobNi1PT3sNH5KIZEB+WaNZaTkK8gAzwvx7V+V0GaDD/P7L5Qa4mRIsXa1xDxdVM7ne4B5zEK1KCePzbZJuv769IbNgzIqAfELJWmXE=
+  - secure: Axr+U8LHOxmGi8GyeL0pP0+84ucxeET59BUpHgANgro+2jLwoLzpvssguc0RHxftVsQBiz2CFfFBVIR6ol+Hf6yjI310Yqw9znBvJuxO0EbAGclGkWyx8DdoWlCSwz+RgiIfIoa7E6QZWt+CZwFbi3vTFutl6NX5Reu0wdpHZlfLT8xkWP3PP7awgL2g75WsNKNv4A1CvzkF3j/hi6JU/b3QsBMTOCHA1vcdpKhjF8MiS9d7gzdwsTr6MoDGVab/oGVCljiFmIPYTa4+xp/q1MYkSizxBmq1mqjXkn1RrZP25QGqgubfgtiywk8H9kDE8/POPJuoteO2gnz2u9I1AkVrrsxCfU9XJzXXwVfbFxVlyOIfC9gZj5ZI9niOqTdytPUMPOzJw79xUvYMeoS7ydcrm3O8gCs0/fBnSH0KfMq37um8zE8zah90M/8CcJjS5KC8hFO/tZomkUASoTwnJEZh4OdSG8Ys9YnKMKBDCuI8HUR2zlVPrv/8ACO4eSZ4LCaNeXQVWil5NNFgLPzvkElqpywVmQaGY3UD2W2w76AEs5sF4n/naQsi6L6oy61mDP8j+wSooQczx/1enue/hzjxZUles7p3ek0yha6N6fxkt/MyvEYWmw98wQZmsBzEahTGXy9iWfobFu0jUzUPXhSdoMul3Q9fceUSP9f+XV8=
+  - secure: QdhoQVjffAiAPcUjN4oPUPuWJX3w7wJQS+p1VE3HWjMGWEFxlEd4k1F91L7uTcibbaogSCZLmh6gbLlcykS9Hi7rD+eiarAlH7acitpuOtTOctrqmx3TmaHV+2Z5aBj33vhA7Ug7/Rmi6JJa83z1jT1AYz1bggDE+ybUp2vQB9NWcuvXBCf6gFnVrYmiCUeJsCQmZubJCvDxxXjNhw9QfpVDCnPXilBQtzKx5qWIx+Ir8kOxTP8xucaqi6VD+rZtT3ifSEZ4suWS68+ZI9AreDAhIaArI5Pmdv/KWaum2LVfeIvxt6lMNOUpoj+83yilapPSoQS6cgOtEmdr7U8ZMVdueA9Nada96BjRErrGNYMcAglMHrNpFfYUVj8CdqlN9IlAQyr7rMz9xWKweCexFTv7bOB6rr38bscZBbhDzo9Iswlr/3UnYeV8VoVXevSqK1+eLFJdvAMR04hT4wa2g1vsujbO/SX2MoEeEiGkOG3Utb5x4d/iHoSQ0vy17vNZ1KCciUpKlWqh8MxkXwWGFYUNilEr9P0SeOU0Qt/8DRzzTQ0p9+j4ihlpDXFk2nv9HBMdQdVpoNqhPyNH2uxSIkuzizrWQtId13MtrdlFEVdpiTjr9OVMra+KuHlUfNtuPU745xhES/SpcLrDH1RZWiBdBPaLPg6r7UFKem1q3oM=
```

Comment:
I'm curious; why are there three secrets here?

Reply:
Hey James, the third one is `ARTIFACTS_BUCKET` but, that's not really needed being encrypted so, moved it to the normal `env` section of the job. Done with 95abaee.

Let me know if you're interested in the other two too. Thanks!